lib/src/screens/dashboard/wallet_menu.dart

```diff
@@ -52,6 +52,22 @@ class WalletMenu {
           image: Image.asset('assets/images/open_book_menu.png',
               height: 16, width: 16),
 	  handler: () => Navigator.of(context).pushNamed(Routes.addressBook)),
+      WalletMenuItem(
+        title: S.current.backup,
+        image: Image.asset('assets/images/restore_wallet.png',
+          height: 16,
+          width: 16,
+          color: Palette.darkBlue),
+        handler: () {
+          Navigator.of(context).pushNamed(
+            Routes.auth,
+            arguments: (bool isAuthenticatedSuccessfully, AuthPageState auth) {
+              if (isAuthenticatedSuccessfully) {
+                auth.close();
+                Navigator.of(auth.context).pushNamed(Routes.backup);
```

Comment:
this will break with the new flushbar package
if this will be merged first, then we will modify it in CW 186
else use `auth.close(route:Routes.backup)` instead after CW-186 is merged